frontend/public/components/utils/owner-references.tsx

```diff
@@ -10,6 +10,7 @@ export const OwnerReferences: React.FC<OwnerReferencesProps> = ({ resource }) =>
       kind={referenceForOwnerRef(o)}
       name={o.name}
       namespace={resource.metadata.namespace}
+      owner={o}
```

Comment:
The problem with this approach is that every single place in console we have `ResourceLink`, we need to pass `owner` now. That would be an enormous change :(

I would skip this for now, and we can try to address `ResourceLink` in a separate PR.

Reply:
Yeah, I couldn't figure out a way to do it without adding this information. I'll refactor all the logic back out.

Reply:
Do you still want the owner reference to point to the same link as the managed by link? Would it be ok to generate the link and pass it to resource link? Or would you prefer a custom approach (resource icon and link)?

Reply:
> Do you still want the owner reference to point to the same link as the managed by link? Would it be ok to generate the link and pass it to resource link? Or would you prefer a custom approach (resource icon and link)?

You don't want to change the owner reference link. You want it to point at the actual owner reference, not the CSV. The managed by link is still available in the header.

Reply:
Let's deal with trying to link to the operand details page as a separate PR.